public/js/app.js

```diff
@@ -31022,6 +31026,29 @@ $(".track-activity-chart").each(function(index, element) {
     }
   });
 
+  localStorageHasKey = function(key) {
+    return localStorage.getItem(key) !== null;
+  };
+
+  loadCommentFromStorage = function() {
+    return $('#submission_comment').val(localStorage.getItem(location.pathname));
+  };
+
+  recordText = function() {
+    return $('#submission_comment').keyup(function(event) {
+      var nitPickText;
+      nitPickText = $(this).val();
```

Comment:
OK, I feel a bit terrible about picking at this but... I'm going to do it anyway. :)

It's not a "nit pick" it's a "nitpick". But that's actually not where this is going. We're trying to eradicate the word "nitpick" from the site, because some people react really negatively to the word.

So we're going with comment. I think in this case either `text` or `comment` or `body` would be fine.